loopin.go

```diff
@@ -196,19 +207,26 @@ func resumeLoopInSwap(reqContext context.Context, cfg *swapConfig,
 		hash, swap.TypeIn, cfg, &pend.Contract.SwapContract,
 	)
 
-	// Create the htlc.
-	htlc, err := swapKit.getHtlc(swap.HtlcNP2WSH)
+	// Create the htlcs.
+	htlcP2WSH, err := swapKit.getHtlc(swap.HtlcP2WSH)
+	if err != nil {
+		return nil, err
+	}
+
+	htlcNP2WSH, err := swapKit.getHtlc(swap.HtlcNP2WSH)
 	if err != nil {
 		return nil, err
 	}
 
 	// Log htlc address for debugging.
-	log.Infof("Htlc address: %v", htlc.Address)
+	swapKit.log.Infof("Htlc address (P2WSH): %v", htlcP2WSH.Address)
+	swapKit.log.Infof("Htlc address (NP2WSH): %v", htlcNP2WSH.Address)
```

Comment:
If you move this below the initialization of `swap`, you can have a method that logs both and reuse that in `newLoopInSwap`. Als the htlc creation can be moved into that common method, reducing code duplication.

Reply:
ptal